test/pettingzoo/pistonball_continuous.py

```diff
@@ -241,15 +238,15 @@ def reward_metric(rews):
         return rews[:, 0]
 
     # trainer
-    result = onpolicy_trainer(
-        policy,
-        train_collector,
-        test_collector,
-        args.epoch,
-        args.step_per_epoch,
-        args.repeat_per_collect,
-        args.test_num,
-        args.batch_size,
+    result = OnpolicyTrainer(

```

Comment:
forget `.run()`?

Reply:
Done. Sorry, the search-replace was not perfect.. Hopefully with the new structure IDE refactoring functions and mypy will be more useful for preventing copy-paste errors :)